### PR TITLE
Ten-Steps-To-Conquer-A-Hackathon

### PR DESCRIPTION
**Pull request by christophermayfield**
Here's some steps to keep in mind when you attend your first hackathon.

### CODE REVIEW

**Comment by Rafase282** [comment]
Hello @christophermayfield please follow the guidelines already established. You should change the filename and conver the file to a `.md` one.

Guidelines can be found [here](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/PULL_REQUEST_TEMPLATE).

**Comment by alayek** [comment]
@christophermayfield, thanks for the article. I was about to do some Q & A type reviews, mostly proof-reading, grammar etc.; but I guess I have bigger questions that have me occupied.

I think you would understand that this is not like an article on some JS function or the workflow in a particular IDE. The content here, is rather objective; leaving more room for interpretation than a typical subjective mathy article.

To begin with, why 10? What's special about 10? If someone submits an article in future,titled **5 top hacks to a hackathon**, should we include that here as another article? Or should we edit this article by stitching together contents from the new article?

There there is question of authentic resources to bolster the content of a wiki article. These 10 steps are most likely your personal views, and as long as you clearly mention that; it's fine. It's also possible you did a survey or you happen to come across some survey by reliable third-party which lists these 10 (or at least some of these) as success recipes for hackathon.  In that case, please link to such study.

It would also help, if you link to similar external resources (I a guessing these type of articles are out there in plethora).

I think it would be best if you were to submit a [Medium article](https://medium.freecodecamp.com/) along these lines to help our campers. Then FCC can publicize that and spread among our campers.

@abhisekp @Rafase282 @raisedadead what do you guys think?

**Comment by christophermayfield** [comment]
If you think its  more Medium appropriate, I can take it there along those
lines. I definitely understand its not JS related. That is true that it is
anecdotal.

On Saturday, March 26, 2016, Arijit Layek <notifications@github.com> wrote:
> @christophermayfield, thanks for the article. I was about to do some Q &
A type reviews, mostly proof-reading, grammar etc.; but I guess I have
bigger questions that have me occupied.
>
> I think you would understand that this is not like an article on some JS
function or the workflow in a particular IDE. The content here, is rather
objective; leaving more room for interpretation than a typical subjective
mathy article.
>
> To begin with, why 10? What's special about 10? If someone submits an
article in future,titled 5 top hacks to a hackathon, should we include that
here as another article? Or should we edit this article by stitching
together contents from the new article?
>
> There there is question of authentic resources to bolster the content of
a wiki article. These 10 steps are most likely your personal views, and as
long as you clearly mention that; it's fine. It's also possible you did a
survey or you happen to come across some survey by reliable third-party
which lists these 10 (or at least some of these) as success recipes for
hackathon. In that case, please link to such study.
>
> It would also help, if you link to similar external resources (I a
guessing these type of articles are out there in plethora).
>
> I think it would be best if you were to submit a Medium article along
these lines to help our campers. Then FCC can publicize that and spread
among our campers.
>
> @abhisekp @Rafase282 @raisedadead what do you guys think?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub<
https://ci3.googleusercontent.com/proxy/HMgyJg6iCAx5XEf6LJQdYW4V12rb_OFfAJlPnOMcbwwQ7TP0Zru4g_ytXBPkDCBIt8sEErZ8xZWKu5qkBDkpkxwsuI1kTBVqISLZJy8lzWCaGE08-Zkr5sxoGcPeB6nKSPGCK5p_as302g245XTsLPpdC5r_Aw=s0-d-e1-ft#https://github.com/notifications/beacon/AOiSvNKsX_QCBnnZc0EoXBM33u9wub0Zks5pxYcygaJpZM4H4k9C.gif
>

-- 
Christopher Mayfield, BA,
Graduate Student, Psychology
Student Assistant, University Data Center

**Comment by alayek** [comment]
@christophermayfield we accept articles which are not JS related - as long as it's tech related. But at the same time, I don't think we accept articles that are mostly about your experiences with something tech related.

I may be wrong; and if so; others tagged in my previous comment can correct me.

**Comment by christophermayfield** [comment]
Okay, that’s no worries, I didn’t fully read the exclusion criteria.

On Saturday, March 26, 2016, Arijit Layek <notifications@github.com> wrote:
> @christophermayfield we accept articles which are not JS related - as
long as it's tech related. But at the same time, I don't think we accept
articles that are mostly about your experiences with something tech related.
>
> I may be wrong; and if so; others tagged in my previous comment can
correct me.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub<
https://ci5.googleusercontent.com/proxy/JdJmBYI86dMrTfo_U0n_d185FBim0VO-LEpdlnhxf8Ozk541pOpc25qN_E7UO-GtqmO0OwuZLW1lus0cLhyHSsopIDvAo1rWF1eXvWjc8w74zq7odjK453Ms8GCOGGgjgz9gS2fp3pmCx7i7zZq961mbN3pLjQ=s0-d-e1-ft#https://github.com/notifications/beacon/AOiSvK86wml2D_T3DWMo9Xq8sGPRP7F-ks5pxYiMgaJpZM4H4k9C.gif
>

-- 
Christopher Mayfield, BA,
Graduate Student, Psychology
Student Assistant, University Data Center

**Comment by Rafase282** [comment]
@alayek you are correct, tech related we take as long as it is an informational article not in first person. If you want to tell about the problem you had with something, you could create a "Common problem with xxx" article stating what they are and how to deal with them.

**Comment by raisedadead** [comment]
@christophermayfield 

Firstly, this is a good write up. 

Now, coming to the criteria for wiki:
* We do not have any hard and fast rules for the content except the being related to Tech.
* A wiki article should not be in first person narrative.
* A wiki article is a reference point for campers to **"Get Started"** when they need some quick help, its the logical place to visit to learn tricks and tips that help them in some way or the other.
* Its not a place to write up a personal experience.

Your article, except for being in first person, actually is erring on the border of being a blog post and wiki article, If you could change the narrative to be similar to our other wiki articles, perhaps this can be accepted at discretion of the others.

@alayek regarding your query for 
> To begin with, why 10? What's special about 10? If someone submits an article in future,titled 5 top hacks to a hackathon, should we include that here as another article? Or should we edit this article by stitching together contents from the new article?

We should be taking the best out of both in such a scenario, as we generally do for any topic.

IMHO, we could accept this having changed the narrative and formatting etc.

Thanks

**Comment by alayek** [comment]
@christophermayfield did you decide what you want to do? Whether to take the content to Medium or format the article's language to suit wiki style?

**Comment by christophermayfield** [comment]
I’ll format the articles language to suit wiki style. I’ll give it a go and
see what you guys think.

On Thursday, March 31, 2016, Arijit Layek <notifications@github.com> wrote:
> @christophermayfield did you decide what you want to do? Whether to take
the content to Medium or format the article's language to suit wiki style?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub<
https://ci3.googleusercontent.com/proxy/0MaN9y1QeJWaHOyvIV1829-2EVZh8DpHYsoE3Q2cMPuZTMTF7_hF_zTclnL3mRTBmGqtm8Yi91zwk5wWn73crfa-eFYsxUK8MkOhEacFFCRx5X45bdC856UM-MGoHQ7yN4uYWEgQeSjeZHg6VHsY9jNFrDHUJQ=s0-d-e1-ft#https://github.com/notifications/beacon/AOiSvA6cMeeJ3RhVQ-GvkmKkgke3wnAjks5pzIZjgaJpZM4H4k9C.gif
>

-- 
Christopher Mayfield, BA,
Graduate Student, Psychology
Student Assistant, University Data Center

**Comment by SeanFCC** [comment]
@alayek 

>To begin with, why 10? What's special about 10?

There are already at least two articles in the wiki I believe that are begun with a 'this many ways' type of subject.

**Comment by christophermayfield** [comment]
Okay, I’ll take it to medium then, sorry.

On Saturday, April 2, 2016, Sean <notifications@github.com> wrote:
> @alayek
>
> To begin with, why 10? What's special about 10?
>
> There are already at least two articles in the wiki I believe that are
begun with a 'this many ways' type of subject.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub<
https://ci6.googleusercontent.com/proxy/v8SVUK77-aJhiCsem1eGLH5i3YDSGC7T5tcT4Yhl6TwGSTfEqQHj1V_GkgpfS9cRdKonSDpryZjWmHZrAirFPNlI_kxJD6AOApdBiU_HeifD5bD1ftYEuw4mKayLB6Nj8yZHpRdkkYVnU9uY6rOOldM0qC2jkA=s0-d-e1-ft#https://github.com/notifications/beacon/AOiSvLev6xIjmEFq2rI-td1NnR2TjMMTks5pztNkgaJpZM4H4k9C.gif
>

-- 
Christopher Mayfield, BA,
Graduate Student, Psychology
Student Assistant, University Data Center

File: 10-steps-to-conquer-a-hackathon.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  you may be left out. 
+  
+2. Don't be afraid 
+
+	 I recently competed in my first hackathon, and I was completely scared with a major case of imposter syndrome. 
+  The key for hackathons is to just got for it without any fear, you are indeed a great asset to a team. 
+
+3. Just keep pushing 
+
+	If you are at a hackathon and your project comes at a hault, just keep pushing, part of a project is better
```
Comment (theflametrooper) [inline 10-steps-to-conquer-a-hackathon.md]: comes **to** a hault...

File: 10-steps-to-conquer-a-hackathon.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  
+4.  Enjoy the freebies 
+	
+	As stressful as hacking away at something you're creating can be, there is alot of cool stuff that some of the sponsors of 
+  hackathons give away, such as shirts, stickers, and occasionally, cool gadgets. Stock up!
+  
+
+5. Enjoy the free food
+
+	Another awesome aspect of hackathons is free food, since you're there for minimum of 12 hours up to 4 or 5 days (extrme cases), they provide free meals for the participants, YUM!
```
Comment (theflametrooper) [inline 10-steps-to-conquer-a-hackathon.md]: extrme should be **extreme**

File: 10-steps-to-conquer-a-hackathon.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+6. Learn as much as you can
+  
+  	One of the best parts about a hackathon is learning some really cool things. A lot of people who come to the events have different
+  skill sets, expertise, and educational background. From this, you can learn anything and everything they have to offer. Some of the 
+  participants will give you links to cool libraries, tutorials and communities, some will just throw around a new buzzword, 
+  some will even show you a cool ergonomic gadget that can improve your workflow. Last but not least, learning some technical skills on the spot with other participants is invaulable!
+  
+  
+7. Network
+
```
Comment (theflametrooper) [inline 10-steps-to-conquer-a-hackathon.md]: Capital C for **Coming**

File: 10-steps-to-conquer-a-hackathon.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  
+  	One of the best parts about a hackathon is learning some really cool things. A lot of people who come to the events have different
+  skill sets, expertise, and educational background. From this, you can learn anything and everything they have to offer. Some of the 
+  participants will give you links to cool libraries, tutorials and communities, some will just throw around a new buzzword, 
+  some will even show you a cool ergonomic gadget that can improve your workflow. Last but not least, learning some technical skills on the spot with other participants is invaulable!
+  
+  
+7. Network
+ 
+ 	coming from a non-technical educational background, this was one of the most important and eye opening aspects for me at my first hackathon. Often times, recruiters will be there,  other developers, VCs, and even lawyers looking to represent the next big innovation. Finally, you can make friends for life with some of the other attendees either in your group or not.
```
Comment (theflametrooper) [inline 10-steps-to-conquer-a-hackathon.md]: Often **at** times

File: 10-steps-to-conquer-a-hackathon.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+8. Go at it alone
+  
+  
+  	This, initally was one of my biggest mountains of fear to climb. I really wanted to go with some people that I know, but if you do that, they become a crutch and you cling to them. The goal is to branch out and make new friends and meet new people. You'll be glad that you did. 
+
+
+9. Embrace the experience
+
+	Sometimes when you're caught in the middle of a bug you can't fix, or your team seems to be not getting along, remember that 
+  all of this is part of the experience of coming to a hackathon. You're not the first person to feel that way and you won't be the
```
Comment (theflametrooper) [inline 10-steps-to-conquer-a-hackathon.md]: that way should be **this way**

File: 10-steps-to-conquer-a-hackathon.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+  last. Treat it all as part of the learning experience.
+  
+    
+10. Have fun!
+
+	One one my favorite quotes from an anonymous source is "if you're not having fun, you're doing it wrong". Make friends, step out
+  of your comfort zone, emjoy the sleepless night, drink some caffeine, laugh when you get a bug, and try to fix it, you'll remember
```
Comment (theflametrooper) [inline 10-steps-to-conquer-a-hackathon.md]: emjoy should be **enjoy**

**Comment by alayek** [comment]
@christophermayfield I feel I have been unnecessarily critical of your contribution earlier. This is a well thought-out article. And I have no qualms with the subject, nor the way you are presenting it.

However, please fix what @theflametrooper suggested. Then we are good to go.

**Comment by Rafase282** [comment]
Please change the filename from `10-steps-to-conquer-a-hackathon.md` to `10-Steps-To-Conquer-A-Hackathon.md` or better yet `Conquer-Hackathons.md`

The number of steps could change in the future, but after creation the less renaming the better for linking purposes. The inside title could be the 10 steps ... as the original.

**Comment by theflametrooper** [comment]
Thanks @Rafase282 for fixing those spelling mistakes.

**Pull request merged**